packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -26,8 +26,11 @@
   "force_debugger_stop_text": "SFDX: Stop Apex Debugger Session",
   "force_config_list_text": "SFDX: List All Config Variables",
   "force_alias_list_text": "SFDX: List All Aliases",
+  "force_org_delete_default_text": "SFDX: Delete Default Org",
+  "force_org_delete_username_text": "SFDX: Delete Org...",
   "force_org_display_default_text": "SFDX: Display Org Details for Default Org",
   "force_org_display_username_text": "SFDX: Display Org Details...",
+  "force_org_list_clean_text": "SFDX: Remove All Expired Orgs",
```

Comment:
Do we have a character limit, either for real or just for what looks good? 
SFDX: Remove All Deleted and Expired Orgs - is this too long?
SFDX: Remove Deleted and Expired Orgs (thinking all may not be necessary, implied?)

Reply:
I'll go with `SFDX: Remove Deleted and Expired Orgs`